app/models/concerns/foreman/thread_session.rb

```diff
@@ -70,10 +70,16 @@ def current=(o)
           end
 
           if o.is_a?(User)
-            Rails.logger.info "Current user: #{o.login} (#{o.admin? ? 'administrator' : 'regular user'})"
+            user = o.login
+            type = o.admin? ? 'admin' : 'regular'
+            if o.internal?
```

Comment:
Actually, you want to check on `hidden?`, as internal means it's not coming from LDAP.